src/test/java/io/inscopemetrics/kairosdb/aggregators/HistogramMeanAggregatorTest.java

```diff
@@ -73,4 +77,23 @@ public void testAggregator() {
 
         assertFalse(result.hasNext());
     }
+
+    @Test
+    public void testOverflow() {
```

Comment:
nit, bikeshedding: Maybe `testCountDoesntOverflow`

Reply:
Lol. Yeah probably. It was written originally to prove it did overflow ;)